examples/debugging/simple_spread/feedforward/decentralised/run_madqn.py

```diff
@@ -84,7 +84,6 @@ def main(_: Any) -> None:
         exploration_scheduler_fn=LinearExplorationScheduler(
             epsilon_start=1.0, epsilon_min=0.05, epsilon_decay=5e-4
         ),
-        importance_sampling_exponent=0.2,
```

Comment:
Yes this was for PER. And I took it out for the time being because I was not convinced that it was done in a robust way. I think in future we can look at re-implementing it.